.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
There are [almost 500 issues without labels](https://github.com/OrchardCMS/OrchardCore/issues?q=is%3Aopen+is%3Aissue+no%3Alabel), because we didn't enforce the very rudimentary task of making the author select if their issue is a bug or feature request. This makes managing those issues a lot harder.

I'd actually also enforce the templates too. Currently, you can just remove the template and write into the issue description whatever. This requires the one replying to the issue ask the same questions.